torchgeo/datasets/idtrees.py

```diff
@@ -567,25 +566,25 @@ def normalize(x: Tensor) -> Tensor:
 
         return fig
 
-    def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
+    def plot_las(
+        self, index: int
+    ) -> "pyvista.Plotter":  # type: ignore[name-defined] # noqa: F821
```

Comment:
What was the deal with these ignores?

Reply:
The ignores are because we define pyvista inside the `plot_las` method so mypy and flake8 are complaining that pyvista is not defined